pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentIndexCreationDriverImpl.java

```diff
@@ -439,8 +436,8 @@ private void handlePostCreation()
     persistCreationMeta(segmentOutputDir, crc, creationTime);
 
     LOGGER.info("Driver, record read time : {}", TimeUnit.NANOSECONDS.toMillis(_totalRecordReadTimeNs));
-    LOGGER.info("Driver, stats collector time : {}", _totalStatsCollectorTime);
-    LOGGER.info("Driver, indexing time : {}", _totalIndexTime);
+    LOGGER.info("Driver, stats collector time : {}", _totalStatsCollectorTimeNs);
+    LOGGER.info("Driver, indexing time : {}", _totalIndexTimeNs);
```

Comment:
Just noticed that these are logging values in different time units, without specifying the time unit being used. Seems best to unify all three to a single unit, and also specify the time unit in the log statement.